### PR TITLE
bump vcstool to 0.3.0

### PR DESCRIPTION
**Pull request by dirk-thomas**
`vcstool` version dropped support for Python 2.7 and 3.4 hence the removal of `jessie`.

### CODE REVIEW

**Comment by dirk-thomas** [comment]
I will leave it up to the maintainers of this repo (@nuclearsandwich ?) to address the CI failure if you are interested in using the newer version of `vcstool` in the ROS repos. If you have specific suggestions how to address this, I am happy to apply them to this PR.

File: config/vcstool.debian.upstream.yaml
Comment (nuclearsandwich) [inline config/vcstool.debian.upstream.yaml:6]:
```suggestion
filter_formula: Package (% python3-vcstool), $Version (% 0.3.0*)
```

File: config/vcstool.ubuntu.upstream.yaml
Comment (nuclearsandwich) [inline config/vcstool.ubuntu.upstream.yaml:6]:
```suggestion
filter_formula: Package (% python3-vcstool), $Version (% 0.3.0*)
```

**Review by nuclearsandwich — APPROVED** [review]
I added two suggestions to fix the syntax errors leftover from previous reviews when these configs were transformed by hand before being run via aptly.

Since those transformations are now automated and the CI jobs will try the import to verify that source packages are present for all binary packages that's being caught now when it wasn't before.

Thanks for the patch to keep these packages up-to-date. As long as the suggestions look good to you and are committed I'll run the import and update the ROS repositories.

**Comment by dirk-thomas** [comment]
> As long as the suggestions look good to you and are committed I'll run the import and update the ROS repositories.

Applied. CI is still failing though. I am happy to apply any further suggestions.

**Comment by nuclearsandwich** [comment]
> Applied. CI is still failing though. I am happy to apply any further suggestions.

This turned out to be bit rot in the tests of the import script. Resolved in #130 

Since the test of these actual configs has passed it's fine to just merge. Thanks for the update!

**Comment by nuclearsandwich** [comment]
Import jobs

* https://build.ros2.org/job/import_upstream/345/
* https://build.ros.org/job/import_upstream/190/

**Pull request merged**
